src/utils/database.js

```diff
@@ -8,7 +8,7 @@ const GLOBALS = require('../../config/globals');
 const logger = require('../logger');
 
 const configurations = ['test', 'production', 'default', 'migration']; // todo move to config
-const env = process.env.NODE_ENV || 'default';
+const env = process.env.NODE_ENV || 'default'; // TODO cleanup
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected use of process.env.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5246313)